examples/mplot3d/projections.py

```diff
@@ -0,0 +1,55 @@
+"""
+========================
+3D plot projection types
+========================
+
+Demonstrates the different camera projections for 3D plots, and the effects of
+changing the focal length for a perspective projection. Note that matplotlib
+corrects for the 'zoom' effect of changing the focal length.
+
+An increasing focal length between 1 and infinity "flattens" the image, while
+a decreasing focal length between 1 and 0 exaggerates the perspective and gives
```

Comment:
And put the default before this sentence.
```suggestion
An increased focal length between 1 and infinity "flattens" the image, while
a decreased focal length between 1 and 0 exaggerates the perspective and gives
```